packages/lang/Python3/package.mk

```diff
@@ -8,8 +8,8 @@ PKG_SHA256="9e06008c8901924395bc1da303eac567a729ae012baa182ab39269f650383bb3"
 PKG_LICENSE="OSS"
 PKG_SITE="https://www.python.org/"
 PKG_URL="https://www.python.org/ftp/python/${PKG_VERSION}/${PKG_NAME::-1}-${PKG_VERSION}.tar.xz"
-PKG_DEPENDS_HOST="zlib:host bzip2:host libffi:host util-linux:host xz:host autoconf-archive:host"
-PKG_DEPENDS_TARGET="toolchain Python3:host sqlite expat zlib bzip2 xz openssl libffi readline ncurses util-linux"
+PKG_DEPENDS_HOST="zlib:host bzip2:host libffi:host util-linux:host autoconf-archive:host"
+PKG_DEPENDS_TARGET="toolchain Python3:host sqlite expat zlib bzip2 openssl libffi readline ncurses util-linux"
```

Comment:
--enable-xz is in in host and target configure options. Guessing that should be disabled.

Reply:
Cool. I’ll have a look when home. I’m doing the full create_addon build now.

Reply:
I have tidied that up, it had already “not found it” in the Python3 build - but this will stop it in all cases.
```
checking for liblzma... no
checking for lzma.h... no
```